splink/column_expression.py

```diff
@@ -140,6 +140,22 @@ def substr(self, start: int, length: int):
 
         return clone
 
+    def _cast_to_string_dialected(self, name: str, dialect: SplinkDialect):
+        cast_sql = sqlglot.parse_one("cast(___col___ as string)").sql(
+            dialect=dialect.sqlglot_name
+        )
+        return cast_sql.replace("___col___", name)
+
+    def cast_to_string(self):
```

Comment:
you could have a generic `cast_to` function, but it shouldn't be used too much and I think it's easier to type/autocomplete `cast_to_string` rather than provide an arg (and then you get into 'what's the arg - is it `text`, `string` etc.)